### ISSUE TITLE
VAL 43 and VAL 19

### ISSUE THREAD

**Issue by WAWMEN** [molenzwiebel/Deceive#152]
I keep getting VAL 43 and VAL 19 when using deceive for Valorant.

**Comment by dylanriesss** [comment]
same

**Comment by Xellos60** [comment]
Same thing here. When I open deceive, it usually pops up in my system tray at the bottom right. Now it doesnt and when I login into valorant, it gives me an error 43 because deceive is not open when valorant is. I don't have any type of anti-virus or firewall setup. It was working fine last week and I have not downloaded or added anything to my pc since.

**Comment by molenzwiebel** [comment]
I cannot produce this locally right now. Could you give me some more information?

- What OS are you running?
- What version of Deceive are you using?
- If you are ingame and encounter the VAL error, is there a Deceive icon _that you can interact with_ in the tray icons? (make sure to hover over it to ensure that it isn't a ghost icon)
- If you play any other Riot games (e.g. League), does Deceive not work there either?
- Has Deceive never worked for you, or did it only stop working recently?

**Comment by Xellos60** [comment]
1. I’m on windows 10.
2. I’m using deceive 1.11.0 , the latest version.
3. When I get the valorant error, deceives icon is not in the system tray, even before I get the error. When I open deceive it usually pops up in my tray but I noticed that once I use it to open valorant, it closes in my task manager. 
4. I also tried it on league and it did not work.
5. It use to work for me, I used it for months then it stopped working this week and I do not have any firewall or antivirus setup.
